packages/eslint-plugin/src/rules/member-delimiter-style.ts

```diff
@@ -2,6 +2,10 @@ import {
   TSESTree,
   AST_NODE_TYPES,
 } from '@typescript-eslint/experimental-utils';
+import {
+  RuleFix,
+  RuleFixer,
+} from '@typescript-eslint/experimental-utils/src/ts-eslint';
```

Comment:
don't add deep imports on `/src` from plugins.
This is availaible on the `TSESLint` export of `@typescript-eslint/experimental-utils`

Reply:
Fixed here: a27e2e38c955f9211c8f223ce966b289f128e779